src/debiased_spatial_whittle/backend.py

```diff
@@ -1,6 +1,9 @@
+import cupyx.scipy.special
```

Comment:
Should be move inside the get_gamma function to avoid failure when cupy is not installed